src/Components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,16 @@
+import { Heading } from "@chakra-ui/react";
+import { SimpleGrid, Box } from "@chakra-ui/react";
+import { categories } from "../../Database";
+import { Category } from "../../Database/quiz.type";
+import { CategoryCard } from "./CategoryCard";
+import { Header } from "./Header";
+import { QuizList } from "./QuizList";
+
+export function HomePage() {
+  return (
+    <div>
+     <Header />
+     <QuizList/>
```

Comment:
prettier can be used for consistent formatting

Reply:
yup ! must've missed it